API.Tests/Services/LeaderboardServiceTests.cs

```diff
@@ -44,7 +46,8 @@ public LeaderboardServiceTests()
             playerRatingsRepository.Object,
             matchStatsRepository.Object,
             playerRepository.Object,
-            tournamentsService
+            tournamentsService,
+            new Mapper(new MapperConfiguration(configuration => configuration.AddProfile(new MapperProfile())))
```

Comment:
Are you meaning to use the `OsuApiClient` mapper profile here?

Reply:
I don't think so, the mapper here is used [here](https://github.com/osu-tournament-rating/otr-api/blob/9275faf51a161372779589ac507e7f99495bcb84/API/Services/Implementations/PlayerRatingsService.cs#L78):

```cs
Adjustments = mapper.Map<ICollection<RatingAdjustmentDTO>>(currentStats.Adjustments)
```